common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -669,15 +669,14 @@ function(VideoPlayer, i18n, moment, _) {
 
     function setSpeed(newSpeed) {
         // Possible speeds for each player type.
-        // HTML5 =          [0.75, 1, 1.25, 1.5]
+        // HTML5 =          [0.75, 1, 1.25, 1.5, 2]
         // Youtube Flash =  [0.75, 1, 1.25, 1.5]
         // Youtube HTML5 =  [0.25, 0.5, 1, 1.5, 2]
         var map = {
             0.25: '0.75', // Youtube HTML5 -> HTML5 or Youtube Flash
             '0.50': '0.75', // Youtube HTML5 -> HTML5 or Youtube Flash
             0.75: '0.50', // HTML5 or Youtube Flash -> Youtube HTML5
-            1.25: '1.50', // HTML5 or Youtube Flash -> Youtube HTML5
-            '2.0': '1.50'   // Youtube HTML5 -> HTML5 or Youtube Flash
```

Comment:
Not sure I quite understand this change.  What was this doing and why don't we need it anymore?

Reply:
Hm, well I thought it was mapping 2.0 speed to 1.5, but apparently not? /shrug I'll add it back in.

Reply:
OK, this is mapping 2.0 speed to 1.5 if 2.0 isn't available. I'm adding this back in because YT Flash doesn't support 2.0